### ISSUE TITLE
"Build a cube" fails in craftassist

### ISSUE THREAD

**Issue by snyxan** [facebookresearch/droidlet#655]
## Type of Issue

Select the type of issue:
- [ ] Bug report (to report a bug)
- [ ] Feature request (to request an additional feature)
- [ ] Tracker (I am just using this as a tracker)
- [ ] Refactor request
- [ ] Documentation Ask

## Description

Agent failed to build a cube when given the command "build a cube"

## Links to any relevant pastes or documents 

Error log

```
INFO:root:Incoming chats: ['<snyxan> build a cube']
2021-08-11 14:51:11,924 [loco_mc_agent.py:288 - perceive() INFO]: Incoming chats: ['<snyxan> build a cube']
INFO:root:Querying the semantic parsing model
2021-08-11 14:51:11,925 [nsp_model_wrapper.py:65 - query_for_logical_form() INFO]: Querying the semantic parsing model
Error validating:
{'dialogue_type': 'HUMAN_GIVE_COMMAND', 'action_sequence': [{'action_type': 'BUILD', 'schematic': {'filters': {'triples': [{'pred_text': 'has_name', 'obj_text': [0, [2, 2]]}]}, 'text_span': [0, [2, 2]]}}]}

Additional properties are not allowed ('triples' was unexpected)

Failed validating 'additionalProperties' in schema[1]['properties']['schematic']['properties']['filters']:
    {'additionalProperties': False,
     'properties': {'comparator': {'$ref': '#/definitions/comparator'},
                    'contains_coreference': {'const': 'yes'},
                    'memory_type': {'enum': ['TASKS',
                                             'REFERENCE_OBJECT',
                                             'CHAT',
                                             'PROGRAM',
                                             'ALL']},
                    'output': {'$ref': '#/definitions/output'},
                    'selector': {'$ref': '#/definitions/selector'},
                    'where_clause': {'$ref': '#/definitions/where_clause'}},
     'type': 'object'}

On instance['schematic']['filters']:
    {'triples': [{'obj_text': [0, [2, 2]], 'pred_text': 'has_name'}]}
ERROR:root:Invalid parse tree for command 'build a cube'

2021-08-11 14:51:13,890 [nsp_querier.py:170 - get_logical_form() ERROR]: Invalid parse tree for command 'build a cube'

ERROR:root:Parse tree failed grammar validation:
 {'dialogue_type': 'HUMAN_GIVE_COMMAND', 'action_sequence': [{'action_type': 'BUILD', 'schematic': {'filters': {'triples': [{'pred_text': 'has_name', 'obj_text': [0, [2, 2]]}]}, 'text_span': [0, [2, 2]]}}]}

2021-08-11 14:51:13,890 [nsp_querier.py:171 - get_logical_form() ERROR]: Parse tree failed grammar validation:
 {'dialogue_type': 'HUMAN_GIVE_COMMAND', 'action_sequence': [{'action_type': 'BUILD', 'schematic': {'filters': {'triples': [{'pred_text': 'has_name', 'obj_text': [0, [2, 2]]}]}, 'text_span': [0, [2, 2]]}}]}

ERROR:root:Returning NOOP
2021-08-11 14:51:13,890 [nsp_querier.py:173 - get_logical_form() ERROR]: Returning NOOP
INFO:root:Sending chat: I don't know how to answer that.
2021-08-11 14:51:14,620 [craftassist_agent.py:305 - send_chat() INFO]: Sending chat: I don't know how to answer that.

```

## Checklist

Use this checklist if you are using this issue as a tracker:

- [ ] Task 1
- [ ] Task 2
- [ ] Task 3 ...

**Comment by aszlam** [comment]
This is a mismatch between current semantic parser and updated spec... will retrain next couple of days

**Comment by aszlam** [comment]
should be fixed in #657
